libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts

```diff
@@ -257,7 +273,7 @@ export class SkyInlineFormComponent implements OnInit, OnDestroy {
         break;
     }
 
-    return promise;
+    return promise; // uhhhh not sure how this was working before?? needs to be duplicated across each case statement... right? then needs to have an else statement?
```

Comment:
Lets touch base and chat about this. I think I see how we can make this better.

Reply:
created a separate bug for this. will do in a separate PR